easy_phi/app.py

```diff
@@ -124,11 +126,11 @@ def get(self):
 
 # URL schemas to RequestHandler classes mapping
 APPLICATION = tornado.web.Application([
-    (r"/", VersionHandler),  # Default page
-    (r"/version", VersionHandler),
-    (r"/modules", ModulesListHandler),
-    (r"/pick_module", SelectModuleHandler),
-    (r"/send_command", SCPICommandHandler),
+    (r"/", PlatformInfoHandler),  # Default page
```

Comment:
It is better to implement PICK modules through modules handler by specifying particular port ID. Same with send_command. Please refer to document for that  https://docs.google.com/document/d/17jka7XUzQt5636tGPPW6-f6fDyBZjouqWoKAbQNsXdE/edit#